src/collections/_documentation/clients/go/context.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Context
+sidebar_order: 3
+---
+
+All of the `Capture*` functions accept an additional argument for passing a `map` of tags as the second argument and context data as remaining ones.
+Tags in Sentry help to categories and give you more information about the errors that happened, where other context data is more "environment" specific.
```

Comment:
Small edits. "Tags in Sentry help categorize data and give you more information about the errors that happened, where other context data is more "environment" specific. (NOTE: direct low-level Capture call accepts the second argument only - tags, without additional interfaces)."